### DISCUSSION TITLE
Delete Account Permanently

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22112] [New to GitHub]
<p>I am having a very difficult time trying to delete my account. I have been trying to locate information on how to delete my account but I’m not having any luck finding the instructions to tell me what I have to do to permanently delete the account and I have been searching several different actions within the settings and other locations that I searched hoping that I would have found the correct directions or actions that I would have to complete in order to successfully delete my account permanently but I’m not having any luck whatsoever. PLEASE HELP ME WITH THIS ISSUE AND FINALLY COMPLETE THE PROCESS OF PERMANENTLY DELETING MY ACCOUNT!!!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Hi! Sorry to see you go, but we have instructions here:</p>
<p><a href="https://help.github.com/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account" class="onebox" target="_blank">https://help.github.com/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account</a></p>

**Reply by 01201255931** [reply]
الحمايه على حسابي وعلى الجيميل من اي هكر

**Reply by EA1IP** [reply]
Ruego la baja definitiva de GitHub por no serme de ninguna utilidad.
Gracias.

**Comment by ghost** [comment]
I have my own instructions..I want all accounts with my name and info permanently deleted..my identity was stolen may 2022..these people set up online buissnesses events and debit cards bank accounts they got a digital passport in my name and believe it or not houses on Zillow by proxy..it has been a nightmare and I had enough..it's not me.never was .thank u

**Reply by ghost** [reply]
I have same thing going on and can’t delete this damn thing I didn’t join

**Reply by ghost** [reply]
Nobody helps us here

**Comment by akashdip2001** [comment]
I understand your concern. To delete your GitHub account, you can follow these steps:

1. **Log in to GitHub:**
   - Make sure you are logged in to the GitHub account you want to delete.

2. **Account Settings:**
   - Click on your profile picture in the top-right corner.
   - Choose "Settings" from the dropdown menu.

3. **Account Security:**
   - In the left sidebar, click on "Account security."

4. **Account Deletion:**
   - Scroll down to the bottom of the page, and you'll find the "Account deletion" section.
   - Click on "Delete your account."

5. **Follow Instructions:**
   - GitHub will guide you through the account deletion process.
   - You may be asked to confirm your password and provide a reason for deletion.

6. **Permanently Delete Account:**
   - Follow the on-screen instructions to complete the account deletion process.

Please note that the process may involve several confirmation steps to ensure account deletion is intentional. After completing these steps, your GitHub account should be permanently deleted.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by djuanny** [reply]
I wanted to close this account and remove my email from it. permanent

**Reply by djuanny** [reply]
I want to close all my accounts and disassociate my personal email from github.com

**Reply by ghost** [reply]
I want all accounts closed with my name and information ..I told u someone is using my identity..I have no buissnesses no events nothing..I am not in no family or team..I am tired of saying this over and over..and Evan and Amber Remillard are using my identity they scammed houses jewelry furniture credit cards  they created a digital passport in my name....they are holding an event coming up and 8m pretty sure using payment methods in my name..close all these accounts in my name please

**Reply by jonathan7767** [reply]
I have followed your instructions and yet when I select delet account, nothing happens! What am I fping wrong & why can I not delete this account?

**Reply by ghost** [reply]
Same I'm so mad

**Comment by ghost** [comment]
how do I find out if there is accounts I don't know about

**Comment by cawzkf** [comment]
como faço para recuperar uma conta excluida hj?

**Comment by Jainam06sheth** [comment]
Pld delete my account

**Comment by dolakit-dev** [comment]
I am facing some issues technical then I need deleting my account

**Comment by AkhileshBandari** [comment]
You can delete your GitHub personal account from the account settings.

Go to **GitHub → Settings → Account**, then scroll all the way down to the **Delete account** section and select **Delete your account**. GitHub will ask you to confirm by entering your username or email and the confirmation phrase shown on screen.

Before doing this, make sure you have backed up anything you want to keep because account deletion is permanent and GitHub cannot restore the deleted content. Your repositories, private forks, wikis, issues, pull requests, and GitHub Pages owned by the account can be removed.

If you don't see the deletion option or GitHub won't let you complete the process, there may be an organization or account-management restriction. In that case, GitHub Support is the appropriate place to handle the account-specific issue.
